content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,148 @@
+requests to a testing Vault CA (this example uses the latest release on
+2019-01-15, you may also use other releases):
+
+    {{< text bash >}}
+    $ export HUB=gcr.io/istio-release
+    $ export TAG=release-1.1-20190115-09-15
+    $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
+    $ helm dep update --skip-refresh install/kubernetes/helm/istio
+    $ ./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
+    $ cat install/kubernetes/namespace.yaml > istio-auth.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-* >> istio-auth.yaml
+    $ helm template --set global.tag=${TAG} \
+        --name=istio \
+        --namespace=istio-system \
+        --set global.hub=${HUB} \
+        --set global.mtls.enabled=true \
+        --set global.controlPlaneSecurityEnabled=true \
+        --set global.proxy.excludeIPRanges="35.233.249.249/32" \
+        --values install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml \
+        install/kubernetes/helm/istio >> istio-auth.yaml
+    $ kubectl create -f istio-auth.yaml
+    {{< /text >}}
+
+The testing Vault server used in this tutorial has the IP
+address `35.233.249.249`. The configuration
+`global.proxy.excludeIPRanges="35.233.249.249/32"` whitelists the IP address of
+the testing Vault server, so that Envoy will not intercept the traffic from
+Node Agent to Vault.
+
+The yaml file `install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml`
+contains the configuration that enables SDS (secret discovery service) in Istio.
+The Vault CA related configuration is set as environmental variables:
+
+    {{< text yaml >}}
+    env:
+    - name: CA_ADDR
+      value: "https://35.233.249.249:8200"
+    - name: CA_PROVIDER
+      value: "VaultCA"
+    - name: "VAULT_ADDR"
+      value: "https://35.233.249.249:8200"
+    - name: "VAULT_AUTH_PATH"
+      value: "auth/kubernetes/login"
+    - name: "VAULT_ROLE"
+      value: "istio-cert"
+    - name: "VAULT_SIGN_CSR_PATH"
+      value: "istio_ca/sign/istio-pki-role"
+    {{< /text >}}
+
+## Deploy a testing workload
+
+This section deploys a testing workload `httpbin`. When the sidecar of the
+testing workload requests for a certificate through SDS, Node Agent will send
+certificate signing requests to Vault.
+
+1.  Download the Istio install package
+(the example here uses [this Istio release package](https://storage.googleapis.com/istio-prerelease/daily-build/release-1.1-20190115-09-15/istio-release-1.1-20190115-09-15-linux.tar.gz))
+and decompress it:
+
+    {{< text bash >}}
+    $ tar xfz istio-release-1.1-20190115-09-15-linux.tar.gz
+    $ cd istio-release-1.1-20190115-09-15
+    {{< /text >}}
+
+1.  Generate the deployment for an example `httpbin` backend:
+
+    {{< text bash >}}
+    $ bin/istioctl kube-inject -f samples/httpbin/httpbin.yaml > httpbin-injected.yaml
```

Comment:
No "bin/"
Surround the input yaml path name with @@. This will result in a link to the file on GitHub.

Reply:
Done, but "surround the input yaml path name with @@" does not result in a link when previewing the page through the localhost:1313 server (perhaps because it is in {{< text bash >}} block?)

Reply:
The @@ syntax is specifically only supported in text blocks. Check out other pages for examples. But it should be:

$ istioctl kube-inject -f @samples/httpbin/httpbin.yaml@ > httpbin-injected.yaml

Reply:
Fixed, thanks!